content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+
+Once you have completed the Ztunnel Networking sub-guide, you will be ready to start setting up the L7 Waypoint Proxy layer in Istio Ambient Mesh.
+
+{{</tip>}}
+
+
+## Introduction 
+
+This guide provides instructions on how to set up and use the L7 Waypoint Proxy layer in Istio Ambient Mesh. It assumes that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh.
+
+L7 traffic routing is based on L4 with the addition of the Waypoint proxy, which is more complex to handle in Envoy. We can also create HPAs to scale it dynamically.
+
+**Overview of the Layer 7 Waypoint proxy**
+
+TODO
+
+**Benefits of using the Layer 7 Waypoint proxy**
+
+TODO
+
+## Pre-requisites & Supported Topologies
+
+Before you begin, make sure that you have the following prerequisites in place:
+1. Istio Ambient Mesh installed and configured
+2. Ztunnel proxy is installed and running
+3. Mutual TLS (mTLS) enabled and configured
+
+## Understanding the L7 Waypoint Proxy Default Configuration
+
+<< Consider breaking this out into bullets for easier reading TODO >>
+
+The L7 Waypoint Proxy layer in Istio Ambient Mesh is designed to provide a usable configuration out of the box with a fixed feature set that does not require much, or any, custom configuration. Currently, there are no configuration options that need to be set other than the `waypoint` profile setting. Once this profile is used, this in turn sets two internal configuration parameters within the Istio Operator which eventually set the configuration of the L7 Waypoint Proxy.
+
+In the future, there may be some additional limited configurability for L7 Waypoint Proxies. However, for now, the following are all configured with fixed default configurations that are not customizable:
+
+* Networking between pods and L7 Waypoint Proxies
+* Networking between L7 Waypoint Proxies
+* Networking between L7 Waypoint Proxies and sidecar proxies
+
+#### Additional Notes
+
+* The L7 Waypoint Proxy layer can coexist with sidecar proxies in the same cluster. This allows you to use the L7 Waypoint Proxy for services that require L7 functionality and sidecar proxies for services that do not.
+* The L7 Waypoint Proxy is implemented using Envoy. This means that you can use all of the features of Envoy in your L7 Waypoint Proxies.
+* The L7 Waypoint Proxy is still under development, but it is already a powerful tool for managing L7 traffic in Istio Ambient Mesh.
+
+For the examples in this guide, we used a deployment of Istio Ambient on a `kind` cluster, although these should apply for any Kubernetes cluster version 1.18.0 or later. Refer to the Getting started guide on how to download the `istioctl` client and how to deploy a `kind` cluster. It would be recommended to have a cluster with more than 1 worker node in order to fully exercise the examples described in this guide. 
```

Comment:
```suggestion
For the examples in this guide, we used a deployment of Istio Ambient on a `KinD` cluster, although these should apply for any Kubernetes cluster version 1.18.0 or later. Refer to the Getting started guide on how to download the `istioctl` client and how to deploy a `KinD` cluster. It would be recommended to have a cluster with more than 1 worker node in order to fully exercise the examples described in this guide. 
```